lib/Dialect/SV/SVDialect.cpp

```diff
@@ -41,3 +51,99 @@ SVDialect::SVDialect(MLIRContext *context)
 }
 
 SVDialect::~SVDialect() {}
+
+//===----------------------------------------------------------------------===//
+// Name conflict resolution
+//===----------------------------------------------------------------------===//
+
+/// Return a StringSet that contains all of the reserved names (e.g. Verilog
+/// keywords) that we need to avoid for fear of name conflicts.
+static const StringSet<> &getReservedWords() {
+  auto &set = *reservedWordCache;
+  if (set.empty()) {
+    static const char *const reservedWords[] = {
+#include "ReservedWords.def"
+    };
+    for (auto *word : reservedWords)
+      set.insert(word);
+  }
```

Comment:
ManagedStatic does thread safe construction on first "touch", but this use of it isn't.  We need to fill in the keywords as part of the construction of the object.  You can do this by changing:

`static llvm::ManagedStatic<StringSet<>> reservedWordCache;` to use the "object_creator" argument, which is the second template argument to ManagedStatic